video/out/drm_common.c

```diff
@@ -81,8 +81,6 @@ const struct m_sub_options drm_conf = {
             .help = drm_connector_opt_help},
         {"drm-mode", OPT_STRING_VALIDATE(mode_spec, drm_validate_mode_opt),
             .help = drm_mode_opt_help},
-        {"drm-atomic", OPT_CHOICE(drm_atomic, {"no", 0}, {"auto", 1}),
-            .deprecation_message = "this option is deprecated: DRM Atomic is required"},
```

Comment:
No user migration needed, can be directly removed. Or `OPT_REMOVED("this option is removed: DRM Atomic is required")`